transcoder/output/diag/DiagnosticOutputManager.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2022 Google LLC
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import yaml
+
+from transcoder.message import DatacastField, DatacastSchema
+from transcoder.output import OutputManager
+from transcoder.output.exception import OutputFunctionNotDefinedError, OutputManagerSchemaError
+
+
+class DiagnosticOutputManager(OutputManager):
+    """Output manager for representing  messages in diagnostic notation"""
+
+    def __init__(self, schema_max_workers=5, lazy_create_resources: bool = False):
+        self.lazy_create_resources = True
```

Comment:
If not used can both be removed

Reply:
So in this case we wouldn't derive from `OutputManager` for this class? The schema max workers and lazy create resource parameters would still need to be in the signature. This is mostly just ignoring the schema related parameters and short circuiting to the console.

Reply:
In which case call to super init. Otherwise, I think you can eliminate the init and then the init from the base would be used.

Reply:
updated in latest commit